src/plugins/intel_gpu/src/kernel_selector/core/common/jitter.cpp

```diff
@@ -1852,6 +1852,27 @@ std::string FusedOpsCodeGenerator::GetJitLoad(const FusedOpsConfiguration& conf,
 
     bool safe_load = conf.boundary_check == FusedOpsConfiguration::BoundaryCheck::ENABLED;
 
+    // Change JitLoad to ignore LT_ALIGNED_READ LodayType if this input tensor has a linear format(SimpleLayout)
```

Comment:
can't we just use existing code instead of generating the code on our own? What happens if we just change it into LT_UNALIGNED?